package.json

```diff
@@ -84,7 +85,7 @@
     "tmp": "0.0.33",
     "vscode-debugprotocol": "1.6.1",
     "vscode-extension-telemetry": "0.0.15",
-    "yauzl": "2.9.1"
+    "yauzl": "2.10.0"
```

Comment:
Needed to get the newer version of `yauzl` for compatibility with the Razor Language Server zipfiles.